runtime/codert_vm/decomp.cpp

```diff
@@ -36,11 +36,12 @@
 #include "jitregmap.h"
 #include "pcstack.h"
 #include "VMHelpers.hpp"
+#include "OMR/Bytes.hpp"
 
 extern "C" {
 
 /* Generic rounding macro - result is a UDATA */
-#define ROUND_TO(granularity, number) (((UDATA)(number) + (granularity) - 1) & ~((UDATA)(granularity) - 1))
+#define ROUND_TO(granularity, number) OMR::align((UDATA)(number), granularity)
```

Comment:
I'm a little cautious about passing in `UDATA` to `OMR::align` as reading the standard `size_t` is guaranteed to only be at least 16-bit wide. It doesn't seem to imply that `size_t` will be able to contain `UDATA`, as this is an implementation detail. Though AFAIK all of the compilers/systems we use will have `sizeof(size_t) == sizeof(UDATA)`. I'd definitely want to hear @rwy0717 and @keithc-ca opinions on this.

Reply:
I think we should be safe: if `UDATA` is wider than `size_t` we should expect warnings from the compiler about data loss (those warnings should be considered errors).

Reply:
I manually compiled the file with the option `-Wall` enabled. And I don't see any errors or warnings relating to my changes.